emcc.py

```diff
@@ -2808,6 +2808,27 @@ def check_memory_setting(setting):
     # add the default exports (only used for ASYNCIFY == 2)
     settings.ASYNCIFY_EXPORTS += DEFAULT_ASYNCIFY_EXPORTS
 
+    if settings.WASMFS:
+      settings.ASYNCIFY_IMPORTS += ['_wasmfs_opfs_init_root_directory',
+                                    '_wasmfs_opfs_get_child',
+                                    '_wasmfs_opfs_get_entries',
+                                    '_wasmfs_opfs_insert_file',
+                                    '_wasmfs_opfs_insert_directory',
+                                    '_wasmfs_opfs_move',
+                                    '_wasmfs_opfs_remove_child',
+                                    '_wasmfs_opfs_open_access',
+                                    '_wasmfs_opfs_open_blob',
+                                    '_wasmfs_opfs_close_access',
+                                    '_wasmfs_opfs_close_blob',
+                                    '_wasmfs_opfs_read_access',
+                                    '_wasmfs_opfs_read_blob',
+                                    '_wasmfs_opfs_write_access',
+                                    '_wasmfs_opfs_get_size_access',
+                                    '_wasmfs_opfs_get_size_file',
+                                    '_wasmfs_opfs_set_size_access',
+                                    '_wasmfs_opfs_set_size_file',
+                                    '_wasmfs_opfs_flush_access']
```

Comment:
I think maybe you can use wildcard here?

Reply:
Good idea, that should be pretty easy something like `__async`.   Or we could just look for the `async` keyword at the beginning of the function string?

Reply:
I added support for testing if the function is `async` and the `__async` option since that's useful for functions that return a promise but aren't marked async.